app/controllers/manuals_controller.rb

```diff
@@ -6,33 +6,50 @@ class ManualsController < ApplicationController
   before_action :ensure_manual_is_found
   before_action :ensure_document_is_found, only: :show
   before_action :prevent_robots_from_indexing_hmrc_manuals
+  before_action :set_up_education_navigation_ab_testing
 
   def index
     set_expiry(content_store_manual)
 
     render :index, locals: {
-      presented_manual: presented_manual
+      presented_manual: presented_manual,
+      ab_test: ab_test
     }
   end
 
   def show
     set_expiry(document_from_repository)
     document = Document.new(document_from_repository, manual)
+    presented_document = DocumentPresenter.new(document: document)
 
     render :show, locals: {
       presented_manual: presented_manual,
-      document: document
+      presented_document: presented_document,
```

Comment:
My comments on keeping `manual` instead of `previous_manual` apply here too: I don't think there's a compelling reason for the view to know this is a presented document.